CMakeLists.txt

```diff
@@ -547,6 +556,15 @@ if(NOT ASPECT_ADDITIONAL_CXX_FLAGS STREQUAL "")
   message(STATUS "  DEAL_II_CXX_FLAGS_RELEASE: ${DEAL_II_CXX_FLAGS_RELEASE}")
 endif()
 
+
+# Create the combined set of sourc files that make up the project
+set(TARGET_SRC
+    ${ASPECT_SOURCE_FILES}
+    ${ASPECT_MAIN_FILE}
+    ${ASPECT_UNIT_TEST_FILES})
```

Comment:
I think you are now putting source files in a different order:
We move files in UNITY_WB_LAST_FILES and UNITY_LAST_FILES to the end of the list of files, but you have main.cc and unittests after them. I am not sure if this will cause problems or not.

Reply:
Ah, I imagined that the order is really only relevant within the *actual* source files. But I can shuffle the two last categories first.

Reply:
I don't know if this is problematic, but I am sure that you are changing the order. :-)